vnext/local-cli/config/configUtils.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+/**
+ * Search for files matching the pattern under the target folder.
+ * @param {string} folder The absolute path to target folder.
+ * @param {string} filenamePattern The pattern to search for.
+ * @return {array} Return the array of relative file paths.
+ */
+function findFiles(folder, filenamePattern) {
+  const files = glob.sync(path.join('**', filenamePattern), {
+    cwd: folder,
+    ignore: [
+      'node_modules/**',
+      '**/Debug/**',
```

Comment:
nit, should this be also ignoring WinUI3 since that's another config?

Reply:
Sure.